src/views/partials/pagination.hbs

```diff
@@ -31,7 +31,18 @@
                         <li class="page-item"><a href="{{@root.fullUrl}}&page={{.}}" class="page-link">{{ . }}</a></li>
                     {{/if}}
                 {{else}}
-                    <li class="page-item"><a href="{{@root.fullUrl}}?page={{.}}" class="page-link">{{ . }}</a></li>
+                {{#if (portContains @root.fullUrl "3000") }}
```

Comment:
This won't work in production, as the fullUrl  will be something different and probably won't use a port